Gruntfile.js

```diff
@@ -1,3 +1,4 @@
+/*eslint object-curly-newline: ["error", "always"]*/
```

Comment:
Until we can allow no-newline for certain cases without always enforcing it, I think the new lint option should be removed and set back to `always`; there are many times I would want to indent a nested object literal when this new rule would force me to one-line it.

We can always use inline enable/disable eslint comments for object literals we really think should be on one line. Or abandon this enterprise entirely =D